kubernetes/yamls/development/web/ingress-notls.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: web-notls
+  namespace: web
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  rules:
+  - host: staging.loklak.org
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: api-server
```

Comment:
You are pointing this to a service named as `api-server`. But there is no service named as `api-server` in the `web` namespace.